Automattic/wp-calypso
# state(followers): move (email-)followers to redux

#### Changes proposed in this Pull Request

* Migrate `followers` and `email-followers` flux store to redux (using a pre-existing reducer)

#### Testing instructions

1. Go to the people section of a site and choose _Followers_
2. Make sure the list of followers shows up
3. If you scroll down on a page with more than 100 followers you should see another request with incremented `page` param
4. If you switch sites, make sure request params are reset and the list loads fine
5. Try to remove followers, are they deleted from the list and don't appear again on a reload?

Follow the steps above for _Email Followers_ as well.

Note: I left some comments where I would like additional feedback.

Fixes #24218
